core/src/components/toast/toast.tsx

```diff
@@ -177,7 +177,7 @@ export class Toast implements ComponentInterface, OverlayInterface {
     if (this.durationTimeout) {
       clearTimeout(this.durationTimeout);
     }
-    return dismiss(this, data, role, 'toastLeave', iosLeaveAnimation, mdLeaveAnimation, this.position);
+    return dismiss(this, data, role, 'toastLeave', iosLeaveAnimation, mdLeaveAnimation, true, this.position);
```

Comment:
Why does toast need the `true` param but others like alert/action-sheet/etc do not?

Reply:
The last parameter of the `dismiss` utility is optional. Alert/ActionSheet/etc. do not use the last parameter to specify additional options, toast does. 

The new property introduced to `dismiss` has a default value of `true` that matches the base behavior on `main` (removing the overlay element). 

Since Toast specifies a value for the last optional parameter, is has to specify a value for the parameter before it. Alert/ActionSheet/etc. do not, so they do not need to specify the parameter, as it will use the default value.